src/tests/class_tests/openms/source/SpectrumExtractor_test.cpp

```diff
@@ -784,6 +765,71 @@ START_SECTION(extractSpectra())
 }
 END_SECTION
 
+START_SECTION(selectSpectra())
+{
+  ptr->setUseGauss(true);
+  ptr->setGaussWidth(0.25);
+  ptr->setRTWindow(30.0);
+  ptr->setMZTolerance(0.1);
+  ptr->setPeakHeightMin(15000.0);
+  ptr->setPeakHeightMax(110000.0);
```

Comment:
Was there a particular reason for using these min/max peak heights?

Reply:
No, they are just the same values as in pickSpectrum() tests, and that source data is actually different. I could use different values here in selectSpectra()